tests/particles/step-68.with_p4est=true.output

```diff
@@ -0,0 +1,64 @@
+
+DEAL::Number of particles inserted: 24
```

Comment:
You should rename this file to "tests/particles/step-68.with_p4est=true.mpirun=1.output" to avoid subtle problems when running the testsuite.

Reply:
Done, good catch :)!